engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/expression/errors/Syntax.scala

```diff
@@ -147,6 +147,11 @@ object Syntax {
       s"Syntax is not supported yet: $syntaxName"
   }
 
+  case object InconsistentConstructorVisibility extends Reason {
+    override def explanation: String =
+      "All the constructors of a type must be either public or private"
```

Comment:
Not 100%, but I find
```suggestion
      "All constructors of a type must be either public or private"
```
e.g. without _the_ better.